.github/workflows/create-release-transactional.yml

```diff
@@ -301,8 +301,9 @@ jobs:
           mkdir -p swagger-out/transactional-php
           unzip zip/mailchimp-transactional-php.zip -d swagger-out/transactional-php
 
-      - uses: php-actions/composer@v2
+      - uses: php-actions/composer@v5
         with:
+	  php_version: 7.4
```

Comment:
I _think_ this weird line break might confuse the yaml parser? Not 100% sure though

Reply:
oh hmm yeah I just saw that alert come through. I can fix that. Seems odd all the checks passed on the PR?

Reply:
oh I think it might just be a tab?